Fix support for readOnly on all RA Inputs (#9656)
marmelab/react-admin: docs/ReferenceInput.md

<ReferenceInput source="company_id" reference="companies" />
```

## `readOnly`

The `readOnly` prop set to true makes the element not mutable, meaning the user can not edit the control.

```jsx
<ReferenceInput
    source="company"
    reference="companies"
    readOnly
/>
```

> **Warning**: The `readOnly` prop refers to the default [`<AutocompleteInput>`](./AutocompleteInput.md) child. If you pass a child component, you'll have to specify the `readOnly` prop on each input:

Reviewer: ```suggestion
**Warning**: The `readOnly` prop is forwarded to the default [`<AutocompleteInput>`](./AutocompleteInput.md) child. If you pass a custom child component, you'll have to specify the `readOnly` prop on it instead:
```

```jsx
<ReferenceInput source="company" reference="companies">
    <SelectInput readOnly />
</ReferenceInput>
```

## `disabled`

The `disabled` prop set to true makes the default input element not mutable, focusable, or even submitted with the form.

```jsx
<ReferenceInput
    source="company"
    reference="companies"
    disabled
/>
```

**Tip:** Note that `disabled` inputs are **not** included in the form values, and hence may trigger `warnWhenUnsavedChanges` if the input previously had a value in the record. To include the input in the form values, you can use `readOnly` instead of `disabled`.

```jsx
<ReferenceInput
    source="company"
    reference="companies"
    disabled
/>
```

> **Warning**: The `disabled` prop refers to the default [`<AutocompleteInput>`](./AutocompleteInput.md) child. If you pass a child component, you'll have to specify the `disabled` prop on each input:

Reviewer: ```suggestion
**Warning**: The `disabled` prop is forwarded to the default [`<AutocompleteInput>`](./AutocompleteInput.md) child. If you pass a custom child component, you'll have to specify the `disabled` prop on it instead:
```
